src/rp2_common/pico_divider/divider.S

```diff
@@ -11,16 +11,21 @@
 .cpu cortex-m0plus
 .thumb
 
+// PICO_CONFIG: PICO_DIVIDER_DISABLE_INTERRUPTS, Disable interrupts around division such that divider state need not be saved/restored in exception handlers, default=0, group=pico_divider
```

Comment:
There's no "#ifndef ... #define ... #endif" default-value defined for PICO_DIVIDER_DISABLE_INTERRUPTS ?